clients/remote/remote.go

```diff
@@ -57,6 +57,57 @@ func (c Client) Create(ctx context.Context, params *CreateParams) error {
 	return c.printRemote(printRemoteOpts{remote: &res})
 }
 
+type UpdateParams struct {
+	RemoteID         string
+	Name             string
+	Description      string
+	RemoteURL        string
+	RemoteAPIToken   string
+	RemoteOrgID      string
+	AllowInsecureTLS bool
+}
+
+func (c Client) Update(ctx context.Context, params *UpdateParams) error {
+	// build request
+	body := api.RemoteConnenctionUpdateRequest{}
+
+	if params.Name != "" {
+		body.SetName(params.Name)
+	}
+
+	if params.Description != "" {
+		body.SetDescription(params.Description)
+	}
+
+	if params.RemoteURL != "" {
+		body.SetRemoteURL(params.RemoteURL)
+	}
+
+	if params.RemoteAPIToken != "" {
+		body.SetRemoteAPIToken(params.RemoteAPIToken)
+	}
+
+	if params.RemoteOrgID != "" {
+		body.SetRemoteOrgID(params.RemoteOrgID)
+	}
+
+	connection, err := c.GetRemoteConnectionByID(ctx, params.RemoteID).Execute()
```

Comment:
We should be able to get away without this extra GET request. You could make the `AllowInsecureTLS` a `*bool` in the struct so you can distinguish unset from explicit true vs. false. Then at the CLI parsing level you can check `ctx.IsSet(<flag-name>)` to know whether or not you should set the pointer field.

Not as clean as having `urfave/cli` write directly into the `bool` field, but I think it's cleaner.